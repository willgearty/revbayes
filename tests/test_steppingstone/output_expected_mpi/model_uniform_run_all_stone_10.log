Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-30262.26	-30267.88	5.628432	5.215046	0.0907488	0.0390481	0.626367	0.0260988	0.115312	0.0995344	0.128039	0.0893276	0.0789702	0.0111415	0.10103	0.0842765	0.0451331	0.0849974	0.100381	0.190673	0.243324	0.0143106	0.447512	0.050706	0.0567445	0.153649	0.0793299	0.0214737	0.0742715	0.0259174	0.108005	0.174368	0.00147489	0.273043	0.00298235	0.0580133	0.124084	0.00395493	0.0521957	0.121033	0.00815343	0.041488	0.00986036	0.00244694	0.0472304	0.0710971	0.0692935	0.331823	0.106996	0.300436	0.0764829	0.155735	0.0285264	0.510771	0.808937	1.08291	1.59738	0.354264	0.476726	0.0222893	0.146721	0.6855185	4.247042
10	-30750.67	-30754.24	3.566397	0.7139742	0.100704	0.0657596	0.323913	0.0136181	0.14528	0.0572819	0.0457539	0.137901	0.0497351	0.100192	0.200853	0.0923559	0.0135526	0.206924	0.119957	0.0956032	0.180686	0.0541229	0.430972	0.0325254	0.0317165	0.228698	0.125159	0.041095	0.155615	0.0337091	0.248267	0.121326	0.00246333	0.328594	0.00214048	0.131002	0.0363744	0.0043928	0.0600248	0.266438	0.0182429	0.10014	0.00714293	0.00380128	0.0705548	0.0991973	0.0476585	0.33935	0.0584325	0.418437	0.0914884	0.0683598	0.0239329	0.0771753	0.37005	0.929939	2.62284	0.465318	0.31645	0.0579232	0.160308	0.4630045	4.631444
