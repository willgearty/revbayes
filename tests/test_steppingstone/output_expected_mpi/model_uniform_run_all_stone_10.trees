Iteration	Posterior	Likelihood	Prior	psi
0	-30262.26	-30267.88	5.628432	(Galeopterus_variegatus[&index=23]:0.079330,((((Loris_tardigradus[&index=12]:0.084276,(Callicebus_donacophilus[&index=2]:0.039048,Varecia_variegata_variegata[&index=22]:0.153649)[&index=40]:0.002447)[&index=27]:0.108005,(((Chlorocebus_aethiops[&index=5]:0.115312,(Macaca_mulatta[&index=13]:0.045133,Microcebus_murinus[&index=14]:0.084997)[&index=37]:0.008153)[&index=24]:0.021474,((Lepilemur_hubbardorum[&index=11]:0.101030,(Daubentonia_madagascariensis[&index=7]:0.128039,(Cebus_albifrons[&index=3]:0.626367,Hylobates_lar[&index=9]:0.078970)[&index=41]:0.047230)[&index=35]:0.052196)[&index=36]:0.121033,((Perodicticus_potto[&index=18]:0.014311,Galago_senegalensis[&index=8]:0.089328)[&index=32]:0.058013,Nycticebus_coucang[&index=15]:0.100381)[&index=31]:0.002982)[&index=26]:0.025917)[&index=39]:0.009860,(Pan_paniscus[&index=17]:0.243324,Cheirogaleus_major[&index=4]:0.026099)[&index=34]:0.003955)[&index=43]:0.069293)[&index=29]:0.001475,((Propithecus_coquereli[&index=19]:0.447512,Aotus_trivirgatus[&index=1]:0.090749)[&index=42]:0.071097,Colobus_guereza[&index=6]:0.099534)[&index=30]:0.273043)[&index=33]:0.124084,((Tarsius_syrichta[&index=21]:0.056744,Saimiri_sciureus[&index=20]:0.050706)[&index=25]:0.074271,Otolemur_crassicaudatus[&index=16]:0.190673)[&index=28]:0.174368)[&index=38]:0.041488,Lemur_catta[&index=10]:0.011141)[&index=44];
10	-30750.67	-30754.24	3.566397	((Saimiri_sciureus[&index=20]:0.032525,Lemur_catta[&index=10]:0.100192)[&index=25]:0.155615,Galeopterus_variegatus[&index=23]:0.125159,((((Pan_paniscus[&index=17]:0.180686,(Loris_tardigradus[&index=12]:0.092356,(((Propithecus_coquereli[&index=19]:0.430972,Microcebus_murinus[&index=14]:0.206924)[&index=42]:0.099197,((Cebus_albifrons[&index=3]:0.323913,(Nycticebus_coucang[&index=15]:0.119957,(Perodicticus_potto[&index=18]:0.054123,Daubentonia_madagascariensis[&index=7]:0.045754)[&index=35]:0.060025)[&index=32]:0.131002)[&index=31]:0.002140,Hylobates_lar[&index=9]:0.049735)[&index=41]:0.070555)[&index=30]:0.328594,((Galago_senegalensis[&index=8]:0.137901,(Aotus_trivirgatus[&index=1]:0.100704,Lepilemur_hubbardorum[&index=11]:0.200853)[&index=39]:0.007143)[&index=37]:0.018243,(Chlorocebus_aethiops[&index=5]:0.145280,Macaca_mulatta[&index=13]:0.013553)[&index=24]:0.041095)[&index=36]:0.266438)[&index=27]:0.248267)[&index=26]:0.033709)[&index=40]:0.003801,Cheirogaleus_major[&index=4]:0.013618)[&index=34]:0.004393,(Colobus_guereza[&index=6]:0.057282,Callicebus_donacophilus[&index=2]:0.065760)[&index=38]:0.100140)[&index=43]:0.047659,((Varecia_variegata_variegata[&index=22]:0.228698,Tarsius_syrichta[&index=21]:0.031717)[&index=28]:0.121326,Otolemur_crassicaudatus[&index=16]:0.095603)[&index=33]:0.036374)[&index=29]:0.002463)[&index=44];
