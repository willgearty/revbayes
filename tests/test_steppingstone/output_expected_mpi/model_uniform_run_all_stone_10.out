state	power	likelihood
4	1.16658e-302	-29863.4
5	1.16658e-302	-30801.8
6	1.16658e-302	-31694.8
7	1.16658e-302	-29958.6
8	1.16658e-302	-30427.9
9	1.16658e-302	-30454
10	1.16658e-302	-30754.2
